src/frontend-admin/src/components/Group/Group.tsx

```diff
@@ -0,0 +1,25 @@
+import React from 'react'
+import styles from './Group.module.css'
+import cx from 'classnames' //allows use of more than 1 classname
```

Comment:
Nice! I've actually not used this library before - instead I've opted for util functions like this one: https://github.com/mtleonard120/react-ui-components/blob/master/src/utils/concatStyles/concatStyles.ts 

I would say go with it though :) Package looks good.